tests/TripalFieldTestHelper.php

```diff
@@ -0,0 +1,212 @@
+<?php
+/**
+ * This class can be included at the top of your TripalTestCase to facillitate testing
+ * fields, widgets and formatters.
+ */
+class TripalFieldTestHelper {
+
+  // This is for the initialized field, widget or formatter as indicated by
+  // $machine_names in the constructor.
+  public $initialized_class;
+
+  // The name of the class being initialized.
+  public $class_name;
+
+  // Information for the field.
+  public $field_info;
+
+  // Information for the field instance.
+  public $instance_info;
+
+  // The loaded bundle the field is attached to.
+  public $bundle;
+
+  // The entity the field is attached to.
+  public $entity;
+
+  // The type of class we are initializing.
+  // One of 'field', 'widget', or 'formatter'.
+  public $type;
+
+  // The name of the field for the class being tested.
+  public $field_name;
+
+  /**
+   * Create an instance of TripalFieldTestHelper.
+   *
+   * Specifcally, initialize the widget class and save it for further testing.
+   *
+   * @param $bundle_name
+   *   The name of the bundle the field should be part of. This bundle must already exist.
```

Comment:
part of?  How about "attached to?"

```suggestion
   *   The name of the bundle the field should be attached to. This bundle must already exist.
```